docs/source/notebooks/example.ipynb

```diff
@@ -4,16 +4,502 @@
+       "np : integer\n",
+       "    Number of processors used to run job\n",
+       "loglevel : {'CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'}string\n",
+       "    Logging level\n",
+       "\n",
+       "Returns\n",
+       "-------\n",
+       "output : ComplexData:mimetype:`application/x-netcdf`\n",
+       "    Output Netcdf File\n",
+       "\u001b[0;31mFile:\u001b[0m      ~/Desktop/osprey/</home/sangwonl/Desktop/osprey/venv/lib/python3.6/site-packages/birdy/client/base.py-3>\n",
+       "\u001b[0;31mType:\u001b[0m      method\n"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
+    "osprey.full_rvic?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Now that we know which process we wish to run, we can use the docstring to ensure we provide the appropriate parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# run convert\n",
+    "convert_output = osprey.convert(\n",
+    "    convert_config = \"tests/data/configs/convert_local.cfg\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# run parameters\n",
+    "params_output = osprey.parameters(\n",
+    "    params_config = \"tests/data/configs/parameters_local.cfg\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# run convolution\n",
+    "convolve_output = osprey.convolution(\n",
+    "    convolve_config = \"tests/data/configs/convolve_opendap.cfg\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# run full_rvic\n",
+    "full_rvic_output = osprey.full_rvic(\n",
+    "    params_config = \"tests/data/configs/parameters_local.cfg\",\n",
+    "    convolve_config = \"tests/data/configs/convolve_opendap.cfg\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Once the process has completed we can extract the results and ensure it is what we expected."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_tmp_copy(output):\n",
+    "    '''\n",
+    "    Parameters:\n",
+    "        output: http address of the output of an Osprey process\n",
+    "    Return value:\n",
+    "        Path to the copied file in /tmp directory\n",
+    "    '''\n",
+    "    output_content = requests.get(output.get()[0]).content\n",
+    "\n",
+    "    tmp_copy = NamedTemporaryFile(\n",
+    "        suffix=\".nc\", prefix=\"tmp_copy\", dir=\"/tmp\", delete=False\n",
+    "    )\n",
+    "    tmp_copy.write(output_content)\n",
+    "    return tmp_copy.name"
```

Comment:
I think this function can be used in `thunderbird` demo, so it would be nice to be included in `wps-tools`. @nikola-rados what do you think?

Reply:
That is a great idea!